cmd/rpcfuzz/cmd.go

```diff
@@ -38,6 +39,10 @@ var RPCFuzzCmd = &cobra.Command{
 	Short: "Continually run a variety of RPC calls and fuzzers.",
 	Long:  usage,
 	Args:  cobra.NoArgs,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		rpcUrl, _ = flag_loader.GetRpcUrlFlagValue(cmd, false)
+		testPrivateHexKey, _ = flag_loader.GetPrivateKeyFlagValue(cmd, false)
```

Comment:
nit: I know `flag_loader.GetRpcUrlFlagValue(cmd, false)` should not return any error but I would still check if the error returned is `nil`. Just in case the implementation changes in the future.

Reply:
I thought about that and decided to go this way to make it easier to relate to the previous implementation.
This relates directly to the other point about the `required` parameter.